salt/kubernetes-master/init.sls

```diff
@@ -16,7 +16,9 @@ include:
 {% endfor %}
 
 # add some extra names the API server could have
-{% set extra_names = ["DNS: " + grains['fqdn']] %}
+{% set extra_names = ["DNS: " + grains['fqdn'],
+                      "DNS: api",
```

Comment:
Do we want to add a raw `api` here? Maybe it makes sense... not sure if we want to ever refer to the apiserver like this.